manifests/a/app/vleer/1.0.0-beta.3/app.vleer.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: app.vleer
+PackageVersion: 1.0.0-beta.3
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{8905B882-D647-4593-946E-5537499CCFBA}'
+AppsAndFeaturesEntries:
+  DisplayVersion 1.0.0

```

```suggestion
- DisplayVersion: 1.0.0
```
I provided invalid syntax, apologies.